runtime/commonMain/src/kotlinx/serialization/json/JsonExceptionHelpers.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+@file:Suppress("FunctionName")
+
+package kotlinx.serialization.json
+
+import kotlinx.serialization.SerialDescriptor
+
+internal fun JsonInvalidValueInStrictModeException(value: Float) = JsonEncodingException(
+    "$value is not a valid float as per JSON spec.\n" +
+            "You can disable strict mode to serialize such values"
+)
+
+internal fun JsonInvalidValueInStrictModeException(value: Double) = JsonEncodingException(
+    "$value is not a valid double as per JSON spec.\n" +
+            "You can disable strict mode to serialize such values"
+)
+
+
+internal fun jsonUnknownKeyException(position: Int, key: String) = JsonDecodingException(
```

Comment:
Declaration from capital letter clashes with deprecated (compatibility) `typealias JsonUnknownKeyException = JsonDecodingException` since they both have the signature (Int, String)